### PR TITLE
Use heroku_rails_deploy gem

### PR DESCRIPTION
**Pull request by tjwp**
Prime: @jturkel

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ group :production do
 end
 
 group :development do
+  gem 'heroku_rails_deploy', require: false
```
Comment (jturkel) [inline Gemfile]: Should we add `require: false` to the alerts service Gemfile and the `heroku_rails_deploy` README.md?

Comment (tjwp) [inline Gemfile]: Did you want me to make those changes?

Comment (jturkel) [inline Gemfile]: If you have time, that would be awesome.

Comment (tjwp) [inline Gemfile]: Made the changes; didn't bother to PR them.

**Pull request merged**